llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -244,6 +249,42 @@ static const RISCVSupportedExtension SupportedExperimentalExtensions[] = {
 };
 // clang-format on
 
+static const RISCVProfile SupportedProfiles[] = {
```

Comment:
`const` -> `constexpr` to be sure the StringLiteral constexpr constructor gets invoked.